allennlp/tests/common/file_utils_test.py

```diff
@@ -52,19 +53,21 @@ def head_callback(_):
     )
 
 
-@mock_s3
+def set_up_s3_bucket(bucket_name: str = "my-bucket", s3_objects: List[Tuple[str, str]] = None):
+    s3_client = boto3.client("s3")
+    s3_client.create_bucket(Bucket=bucket_name)
+    for filename, key in s3_objects or []:
+        s3_client.upload_file(Filename=filename, Bucket=bucket_name, Key=key)
+
+
+#  @mock_s3
```

Comment:
Having the decorator at the class level had unintended consequences for other tests